src/wrapper-start.js

```diff
@@ -37,20 +37,13 @@
     var newErr;
     if (err instanceof Error) {
       var newErr = new Error(err.message, err.fileName, err.lineNumber);
-      if (isBrowser) {
-        newErr.message = err.message + '\n\t' + msg;
-        newErr.stack = err.stack;
-      }
-      else {
-        // node errors only look correct with the stack modified
-        newErr.message = err.message;
-        newErr.stack = err.stack + '\n\t' + msg;
```

Comment:
I think this might have been here for a good reason.

Currently if I throw errors in Node from this branch, they look like:

```
[Error: ENOENT, open '/Users/guybedford/Projects/es6-module-loader/asdf.js'
	Error loading file:///Users/guybedford/Projects/es6-module-loader/asdf.js]
```

Which looks like an array not a stack.

Any ideas?